task/common/machine/script.go

```diff
@@ -61,6 +68,10 @@ sudo mv terraform-provider-iterative* /usr/bin/tpi
 sudo chmod u=rwx,g=rx,o=rx /usr/bin/tpi
 sudo chown root:root /usr/bin/tpi
 
+curl --location --remote-name https://github.com/iterative/cml/releases/latest/download/cml-linux
+sudo chmod 777 cml-linux
```

Comment:
```suggestion
sudo chmod 755 cml-linux
```

...or this for consistency:

```suggestion
sudo chmod u=rwx,g=rx,o=rx cml-linux
```

💅🏼